README.md

```diff
@@ -1,4 +1,12 @@
 # resurfaceio-logger-go
+
 Logging usage of Golang-based services, with user privacy by design.
 
 Visit <a href="https://resurface.io">resurface.io</a> for general information on usage logging.
+
+## Plan for going about testing
+
+- read over existing test scripts in different resurface loggers
+- implement basic test helpers
+  -profit
+  (https://codeburst.io/unit-testing-for-rest-apis-in-go-86c70dada52d)
```

Comment:
I think our previous readme got deleted but might be beneficial to add our go build, go test commands.